build_library/generate_au_zip.py

```diff
@@ -129,12 +137,12 @@ def DepsToCopy(ldd_files):
       logging.error("ldd for %s failed: %s", file_name, ex)
       sys.exit(1)
 
-  result = _ExcludeBlacklist(list(libs), BLACK_LIST)
-  _EnforceWhiteList(list(libs), WHITE_LIST)
+  result = _ExcludeDenylist(list(libs), DENY_LIST)
+  _EnforceAllowList(list(libs), allow_list=allow_list)
   return result
 
 
-def CopyRequiredFiles(dest_files_root):
+def CopyRequiredFiles(dest_files_root, allow_list):
```

Comment:
Could you add `allow_list` to the Args section in the docstring below?

Reply:
Done